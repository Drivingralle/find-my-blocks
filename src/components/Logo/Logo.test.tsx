import React from "react";
import renderer from "react-test-renderer";
import { Logo } from "./";

jest.mock("./logo.ignore.ts");

it("render correctly", () => {
  const tree = renderer.create(<Logo />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("render pin version", () => {
  const tree = renderer.create(<Logo version="pin" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("render blank if no version is found", () => {
  const tree = renderer.create(<Logo version="raptors" />).toJSON();
  expect(tree).toBe(null);
  expect(tree).toMatchSnapshot();
});
