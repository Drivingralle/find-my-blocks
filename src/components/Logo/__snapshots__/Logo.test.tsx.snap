// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render blank if no version is found 1`] = `null`;

exports[`render correctly 1`] = `
<svg
  fill="#FF4B3E"
  style={
    Object {
      "height": undefined,
      "width": "auto",
    }
  }
  viewBox="0 0 0 0"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    dangerouslySetInnerHTML={
      Object {
        "__html": "<path />",
      }
    }
  />
</svg>
`;

exports[`render pin version 1`] = `
<svg
  fill="#FF4B3E"
  style={
    Object {
      "height": undefined,
      "width": "auto",
    }
  }
  viewBox="0 0 0 0"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    dangerouslySetInnerHTML={
      Object {
        "__html": "<path />",
      }
    }
  />
</svg>
`;
